Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,41 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void SetFilePath_WithNewFile_CheckThatFileIsCreated()
+        {
+            File.Delete("newFile.txt");
+            string newFile = "newFile.txt";
+
+            FileLogger fileLogger = new FileLogger();
+            fileLogger.SetFilePath(newFile);
+
+            Assert.IsTrue(File.Exists(newFile));
+        }
+
+        [TestMethod]
+        public void Log_FileMessageIsCorrect_isTrue()
+        {
+            string newFile = Path.GetTempFileName();
+            string className = "FileLoggerTests";
+            string message = "FileLoggerTests are running";
+
+            LogFactory logFactory = new LogFactory();
+            logFactory.ConfigureFileLogger(className);
```

Comment:
Shouldn't this be passing the file path?
```suggestion
            logFactory.ConfigureFileLogger(newFile);
```